### PR TITLE
Updated README.rst example code

### PR DESCRIPTION
**Pull request by ZiadAmerr**

### CODE REVIEW

**Review by guillesanbri — COMMENTED** [review]
lgtm, one thing tho, the loss when executing this seems to be quite far from the one in the gif, is there any reason for this?

**Comment by ZiadAmerr** [comment]
Yes, the number of training examples and the standard deviation of the error added to the labels

The example here has 3000 feature-label pairs, and the labels have an error added to them with mean of 0 and std of 2. While the graph has about 100 examples only and a std of 0.2. I did that so it would look better in visualization

**Comment by guillesanbri** [comment]
I'd say it should be the same, feel free to change the code if the visualization needs to be simpler imo

**Comment by ZiadAmerr** [comment]
Hey @guillesanbri, it should be much closer now.

**Comment by guillesanbri** [comment]
> Hey @guillesanbri, it should be much closer now.

I'm afraid the final loss is still around ~175 for me :thinking:

File: README.rst [hunk 1]
```diff
@@ -231,29 +232,30 @@ but this can easily be changed to your favorite framework, such as TensorFlow, o
...
+            x = self.linear0(x)
+            x = ivy.functional.relu(x)
+            x = self.linear1(x)
             return x
 
-    ivy.set_backend('torch')  # set backend to PyTorch
+    ivy.set_backend('torch')  # set backend to PyTorch (or any other backend!)
+
+    model = Regressor(input_dim=1, output_dim=1)
+    optimizer = ivy.Adam(0.1)
```
Comment (ZiadAmerr) [inline README.rst:251]:
```suggestion
    optimizer = ivy.Adam(0.3)
```

File: README.rst [hunk 2]
```diff
@@ -231,29 +232,30 @@ but this can easily be changed to your favorite framework, such as TensorFlow, o
...
 
-    model = Regressor(input_dim=3, output_dim=1)
-    optimizer = ivy.Adam(1e-4)
+    x = ivy.random.random_normal(shape=(500, 1), std=10, mean=-1)
+    y = 0.2 * x ** 2 + 0.5 * x + 0.1 + ivy.random.random_normal(shape=(500, 1), mean=0, std=1)
```
Comment (ZiadAmerr) [inline README.rst:254]:
```suggestion
    n_training_examples = 2000
    noise = ivy.random.random_normal(shape=(n_training_examples, 1), mean=0, std=0.1)
    x = ivy.linspace(-6, 3, n_training_examples).reshape((n_training_examples, 1))
    y = 0.2 * x ** 2 + 0.5 * x + 0.1 + noise
```

**Comment by ZiadAmerr** [comment]
Hey @guillesanbri, with the latest code changes, I could reach a loss of 0.01.

File: README.rst [hunk 3]
```diff
@@ -274,9 +278,10 @@ The model's output can be visualized as follows:
 
 .. raw:: html
 
-   <div align="center">
-      <img width="50%" src="https://i.imgur.com/DUc97i2.gif">
-   </div>
+   <video width="320" height="240" autoplay loop>
+      <source src="https://github.com/ZiadAmerr/unifyai.github.io/blob/master/webm/video_hq.webm" type="video/webm">
+      Your browser does not support the video tag.
+    </video>
```
Comment (ZiadAmerr) [inline README.rst:284]:
```suggestion
   <div align="center">
      <img width="50%" src="https://github.com/ZiadAmerr/unifyai.github.io/blob/master/img/regressor_lq.gif">
   </div>
```

File: README.rst [hunk 4]
```diff
@@ -275,7 +279,7 @@ The model's output can be visualized as follows:
 .. raw:: html
 
    <div align="center">
-      <img width="50%" src="https://i.imgur.com/DUc97i2.gif">
+      <img width="50%" src="https://github.com/ZiadAmerr/unifyai.github.io/blob/master/img/regressor_lq.gif">
```
Comment (guillesanbri) [inline README.rst:282]: this should point to an Ivy repo directly :)

**Review by guillesanbri — CHANGES_REQUESTED** [review]
totally missed this sorry! Once the comment is addressed if the code runs fine we can merge it :+1:

**Comment by guillesanbri** [comment]
Hi @ZiadAmerr the example seems to be throwing an error locally for me, at first glance I'd say it's something on the framework side and not really the example, but we should get someone to look into it, could you raise it in Discord? Thanks!

**Pull request merged**